docs/developers/talawa-api/mutations.md

```diff
@@ -2,8 +2,16 @@
 id: mutations
 title: Mutations
 ---
+> Mutations can be classified in a number of ways, including by the type of change that they introduce, the part of the page that they affect, or the manner in which they are introduced. Talawa API introduces mutation to alter it's value to a certain degree.
 
 ## Mutations According to Roles
+<<<<<<< HEAD
+=======
+
+This document outlines our mutations and to which type of user each is constrained. Common types of mutations include insertions and deletions. Insertions are new changes to the Talawa API, while deletions removes.
+
```

Comment:
1. Please fix this. The >>> indicate a merge conflict. @Cioppolo14 FYI
2. This file should be removed from the PR